### PR TITLE
Forecast: Added trigger "forecast"

### PR DESCRIPTION
**Pull request by himanshu0113**
Added "forecast" to the file forecast.pm as trigger.

IA Page: http://duck.co/ia/view/forecast

### CODE REVIEW

**Comment by daxtheduck** [comment]
@himanshu0113 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by MrChrisW** [comment]
Hi @himanshu0113, thanks for your contribution.

>Added "forecast" to the file forecast.pm as directed to me.

Who directed you to add the forecast trigger? 

Currently this IA is triggered internally using deep triggers and doesn't require the addition of new triggers words.

**Comment by himanshu0113** [comment]
Hi @MrChrisW , I am a beginner at open source community. Recently I signed up for DuckDuckGo Hack which is to be held on january 30th.
Danial Davis, community maneger at DuckDuckGo guided me through how to get started and gave me some links for the same. Adding this trigger was one example he gave.

**Comment by moollaza** [comment]
@himanshu0113 Thanks for contributing and welcome to DuckDuckHack!

As @MrChrisW said, this Instant Answer has a very special way of triggering, but there was actually some discussion about adding the word "forecast" as a trigger a while back in https://github.com/duckduckgo/zeroclickinfo-spice/issues/2248

I think given the way the code currently works, we can _try_ this PR out and see how it works.

@himanshu0113 in order for this to get deployed to Beta though, you need to properly link the Instant Answer page to this Pull Request.

Please read http://docs.duckduckhack.com/submitting/pull-request.html to understand the process.

Basically you just need to add the link `https://duck.co/ia/view/forecast` to your initial comment at the top of this page.

**Comment by himanshu0113** [comment]
@moollaza Thanks for guiding me!

I will take care of these things in future and will try to contribute to DuckDuckGo as much as possible.

**Comment by GuiltyDolphin** [comment]
@moollaza Situation with this?

/cc @MrChrisW

**Comment by MrChrisW** [comment]
@GuiltyDolphin Thanks for the bump. @moollaza Needs to deploy this to beta for testing :+1:

File: lib/DDG/Spice/Forecast.pm [hunk 1]
```diff
@@ -5,7 +5,7 @@ use strict;
 use DDG::Spice;
 use Text::Trim;
 
-triggers start => "weather";
+triggers start => "weather","forecast";
```
Comment (GuiltyDolphin) [inline lib/DDG/Spice/Forecast.pm]: Could you add `weather forecast` as well?

**Comment by himanshu0113** [comment]
@GuiltyDolphin Can you please tell me why this commit failed? I just added another trigger.

File: lib/DDG/Spice/Forecast.pm [hunk 2]
```diff
@@ -5,7 +5,7 @@ use strict;
 use DDG::Spice;
 use Text::Trim;
 
-triggers start => "weather";
+triggers start => "weather", "forecast", "weather forecast"
```
Comment (GuiltyDolphin) [inline lib/DDG/Spice/Forecast.pm]: Need a semi-colon ;)

Comment (himanshu0113) [inline lib/DDG/Spice/Forecast.pm]: Just saw that. Updated. Thanks. :P

**Comment by GuiltyDolphin** [comment]
@himanshu0113 Thanks, LGTM :grinning:
